tasks/managed/create-pyxis-image/README.md

```diff
@@ -21,12 +21,18 @@ The relative path of the pyxis.json file in the data workspace is output as a ta
+| orasOptions             | oras options to pass to Trusted Artifacts calls                                                                                                                                                                                                                                                                                                                                                             | Yes      | ""                      |
 | sourceDataArtifact      | Location of trusted artifacts to be used to populate data directory                                                                                                                                                                                                                                                                                                                                         | Yes      | ""                      |
 | dataDir                 | The location where data will be stored                                                                                                                                                                                                                                                                                                                                                                      | Yes      | $(workspaces.data.path) |
 | taskGitUrl              | The url to the git repo where the release-service-catalog tasks and stepactions to be used are stored                                                                                                                                                                                                                                                                                                       | No       | ""                      |
 | taskGitRevision         | The revision in the taskGitUrl repo to be used                                                                                                                                                                                                                                                                                                                                                              | No       | ""                      |
 
+## Changes in 5.0.0
+* Replace `.pyxis.skipLayers` with `.pyxis.includeLayers`
+  * We want to skip the layer details when creating images in Pyxis by default,
+    so to include it in the future, the data will need to set
+    `.pyxis.includeLayers=true` explicitly
```

Comment:
nit pick but maybe `s/includeLayers=true/includeLayers: true/` since that is how it is in the RPA?

Reply:
This is not a json snippet - `.pyxis.includeLayers` is more like what you'd use with `jq` - it's a path in the json. And it is equal to true, so `=true`. If I wanted a json snippet, I would need to include a json snippet:
```
...
data:
  pyxis:
    includeLayers: true
```
Which seems very long...